### DISCUSSION TITLE
Why another quickPreviewContext provider?

### DISCUSSION THREAD

**Discussion by samuelcastro** [UnlyEd/next-right-now#272] [General] [answered]
I saw you added a new `quickPreviewContext` provider and I wonder why? Wouldn't it be easier to just use the `previewModeContext` with a new prop: `isQuickPreviewPage`. 

Btw, what's this `quickPreview` feature to begin with?

I'm just trying to keep things as simple and clean as possible.

**Comment by Vadorequest** [comment, marked as answer]
Quick preview and preview mode are two different things. The quick preview is usable only when preview mode is enabled, but they're different and that's why they have different different hooks/contexts.

The Quick preview feature isn't showcased in the `gcms` preset, because GraphCMS doesn't allow embedding of external iframes, so I couldn't make a demo with it. But it's being used by the `at` preset.

A quick preview is the ability to preview content generated by Next.js from a 3rd party (e.g: a CMS). If you don't need it, you can leave it alone or remove it.

**Reply by samuelcastro** [reply]
Where can I see the quick preview demo? I couldn't find it here: https://v2-mst-aptd-at-lcz-sty.vercel.app/

**Reply by Vadorequest** [reply]
https://nrn.my.stacker.app/login?api_token=be1050d1-de5e-4ae0-97c8-030a132f254b&ref=unly-nrn

Then https://nrn.my.stacker.app/product/view/pro_recFSrY2znI6Z8Dbj 

Currently, the preview is broken as I moved the Next.js page but I'll fix it :)

**Reply by Vadorequest** [reply]
Here's how it looks like:
![image](https://user-images.githubusercontent.com/3807458/105252746-54b3ab80-5b7e-11eb-8162-e8e3a7bacbba.png)


Should be fixed within 10mn.

**Reply by samuelcastro** [reply]
Nice, thanks!
